pkgs/development/python-modules/demjson/default.nix

```diff
@@ -3,13 +3,18 @@
 buildPythonPackage rec {
   pname = "demjson";
   version = "2.2.4";
-  disabled = isPy3k;
 
   src = fetchPypi {
     inherit pname version;
     sha256 = "0ygbddpnvp5lby6mr5kz60la3hkvwwzv3wwb3z0w9ngxl0w21pii";
   };
 
+  checkPhase = stdenv.lib.optionalString isPy3k ''
+    2to3 -w test/test_demjson.py
+  '' + ''
+    PYTHONPATH=build/lib python test/test_demjson.py
```

Comment:
```suggestion
    PYTHONPATH=build/lib ${python.interpreter} test/test_demjson.py
```

Reply:
Thanks for pointing that out. I looked through the nixpkgs documentation and also looked at the environment variables visible from inside `nix-shell -A pypy3Packages.demjson`, and didn't find any way to get the path to the interpreter. It seems to have already been substituted in place in the various bash functions visible in the bash shell.

Reply:
I found that several other expressions under `pkgs/development/python-modules` seem to take `python` as an argument alongside `buildPythonPackage`. I don't know that anything guarantees that they will be called with compatible values for their `python` and `buildPythonPackage` arguments (specifically that the supplied `buildPythonPackage` will use the same Python as what's passed in via the `python` argument), but that seems to be how things are currently done.

I adopted that pattern and tested that this builds on all pypy versions as well.